apps/meteor/app/livechat/imports/server/rest/departments.ts

```diff
@@ -57,10 +57,18 @@ API.v1.addRoute(
 			check(this.bodyParams, {
 				department: Object,
 				agents: Match.Maybe(Array),
+				departmentUnit: Match.Maybe({ _id: Match.Optional(String) }),
```

Comment:
Since we're also allowing users to remove departments from units here, I thought the distinction between `undefined` and `null` could be misleading like this
I mean, in order to remove a department from a unit, users would need to provide `{}` or `{ _id: undefined }`, while if we didn't have an object, just `null` would need to remove the department from its unit. So I felt that having an object would avoid users to perform this action by mistake
wdyt? should we change this?

Reply:
No, I think your idea is fine, just wanted to understand it.